itziakos/zipfile2
# Add a test github actions workflow

fixes #22 

- Remove tox and appveyor testing 
- Add github actions testing
- Remove Python 2.7 testing
  - remove dependency on mock
  - remove compatibility unittest code
- Copy some cpython test.support code to check for symlink support and privileges on windows.
